client/src/areas/party/home/PartyLoginForm.tsx

```diff
@@ -0,0 +1,238 @@
+import * as React from "react";
```

Comment:
This whole file should definitely also be reused from the existing components => issue for later

Reply:
https://github.com/Nasicus/d2-holy-grail/issues/27